addons/mrp/models/mrp_production.py

```diff
@@ -71,11 +71,12 @@ def _get_default_is_locked(self):
         readonly=True, required=True, check_company=True,
         states={'draft': [('readonly', False)]})
     product_variant_attributes = fields.Many2many('product.template.attribute.value', related='product_id.product_template_attribute_value_ids')
+    workcenter_id = fields.Many2one('mrp.workcenter')
```

Comment:
No need to having a stored field, also after thinking twice, maybe adding a comment to explain why this field is created can be useful for the future when we won't remember of these task's details
```suggestion
    workcenter_id = fields.Many2one('mrp.workcenter', store=False)  # Only used by the search
```